jenkins/Jenkinsfile-external

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/groovy
-@Library('test-shared-library') _
+@Library('test-shared-library@mr/ita/68-custom-commit-states') _
```

Comment:
Once `mr/ita/68-custom-commit-states` is merged, use just `@Library('test-shared-library') _`